.travis.yml

```diff
@@ -0,0 +1 @@
+build/ci/travis.yml
```

Comment:
I blindly followed the recommendation of the standard project layout here (to symlink to `build/ci/*`), but is it worthwhile? We don't really need to support multiple CI configurations and it might surprise people.